Create ui-c8y-1021-66-0-date-time-picker-missing-localizations-not-respecting-user-locale-8367-8923.md (#3264)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-66-0-date-time-picker-missing-localizations-not-respecting-user-locale-8367-8923.md

---
date: ""

> title: Date picker now respects user locale and includes missing localizations

Reviewer: ```suggestion
title: Date-time picker now respects user locale and includes missing localizations
```

Reviewer 2: I discussed the terminology with Jan. In the documentation we have date picker, date and time picker, date range picker etc. And we agreed that date picker is the overall term. But if you prefer we can change it here.

> title: Date-time picker now respects user locale and includes missing localizations

Reviewer 2: ```suggestion
title: Date picker now respects user locale and includes missing localizations
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62659
version: 1021.66.0
---

> The date picker component did not respect the user´s locale setting, resulting in missing localizations. This issue has been fixed. Users will now see the date picker in their preferred language and format, providing a more intuitive and localized user experience.

Reviewer: @BeateRixen I think it's worth mentioning that this change applies also to time picker (AM/PM or 24h).
Also along with this change, all other places with date/time will be formatted to current locale.
```suggestion
The date-time picker component did not respect the user´s locale setting, resulting in missing localizations. This issue has been fixed. Users will now not only see the date-time picker in their preferred language and format but every date and time displayed in the UI.
```

> The date-time picker component did not respect the user´s locale setting, resulting in missing localizations. This issue has been fixed. Users will now see the date-time picker in their preferred language and format, providing a more intuitive and localized user experience.

Reviewer 2: ```suggestion
The date picker component did not respect the user´s locale setting, resulting in missing localizations. This issue has been fixed. Users will now see the date picker in their preferred language and format, providing a more intuitive and localized user experience.
```
